torch/csrc/jit/ivalue.h

```diff
@@ -295,6 +361,7 @@ struct IValue {
     at::TensorImpl* as_tensor_impl;
     at::Retainable* as_retainable;
     double as_double;
+    std::string as_string;
```

Comment:
I doubt we want to embed the string in here as is. This means that our small struct (16B) that was meant to represent lightweight values, suddenly gains a 32B member, which means it ends up having 40B. This is a nearly 3x growth in size, which we should avoid. Strings should be probably kept in the `as_retainable` field. They're not meant as a highly performant value anyway, and shouldn't be used too often.